app/localization/translated/be.json

```diff
@@ -971,16 +985,15 @@
   "Notification.failureDefault": "Адбылася памылка пры падключэнні да сервера: {error}",
   "Notification.infoLogout": "Вы выйшлі",
   "Notification.successLogin": "Паспяховы ўваход",
-  "NotificationsTab.updateProjectNotificationsConfigurationSuccess": "Налады апавяшчэнняў былі паспяхова абноўлены!",
   "NotificationCase.attributesLabel": "Атрыбуты (і)",
   "NotificationCase.attributesNote": "Дачылаць апавяшчэнні пра запускі, якія ўтрымліваюць пазначаныя атрыбуты",
   "NotificationCase.controlPanelName": "Правіла",
-  "NotificationCase.always": "Заўсёды",
-  "NotificationCase.failed": "Запуск утрымлівае праблемы",
-  "NotificationCase.more10": "> 10% складнікаў утрымліваюць праблемы",
-  "NotificationCase.more20": "> 20% складнікаў утрымліваюць праблемы",
-  "NotificationCase.more50": "> 50% складнікаў утрымліваюць праблемы",
-  "NotificationCase.toInvestigate": "Запуск утрымлівае складнікі, што патрабуюць аналізу",
+  "NotificationCase.dropdownValueAlways": "Always",
```

Comment:
I thinks it's better to use here translated strings from above

Reply:
Actually, `NotificationCase` strings are a part of someone else changes, but I've updated the translations